lib/jekyll-remote-theme/downloader.rb

```diff
@@ -5,12 +5,12 @@ module RemoteTheme
     class Downloader
       HOST = "https://codeload.github.com".freeze
       PROJECT_URL = "https://github.com/benbalter/jekyll-remote-theme".freeze
-      TYPHOEUS_OPTIONS = {
-        :headers => {
-          :user_agent => "Jekyll Remote Theme/#{VERSION} (+#{PROJECT_URL})",
-        },
-        :verbose => (Jekyll.logger.level == :debug),
-      }.freeze
+      USER_AGENT = "Jekyll Remote Theme/#{VERSION} (+#{PROJECT_URL})".freeze
+      MAX_FILE_SIZE = 1 * (1024 * 1024 * 1024) # Size in bytes (1 GB)
```

Comment:
Do you have any thoughts on whether this should be configurable? Maybe my host’s disks aren’t quite so large?

Reply:
Assume you're talking about the max file size? I put it in largely as a guard against abuse. Not sure how a malicious user might create a never-ending zip since we control the source server, but figured it'd be better to be safe than sorry.